flask_dance/contrib/nylas.py

```diff
@@ -59,9 +62,9 @@ def make_nylas_blueprint(
         client_id=client_id,
         client_secret=client_secret,
         scope=scope,
-        base_url="https://api.nylas.com/",
-        authorization_url="https://api.nylas.com/oauth/authorize",
-        token_url="https://api.nylas.com/oauth/token",
+        base_url=base_url,
+        authorization_url= base_url + "/oauth/authorize",
+        token_url= base_url + "/oauth/token",
```

Comment:
We use f-strings for interpolation across this project, or `base_url.relative("oauth/authorize")` etc.